aws-lambda-scorer/lambda-template/build.gradle

```diff
@@ -5,6 +5,10 @@ dependencies {
     implementation project(':common:transform')
     implementation group: 'ai.h2o', name: 'mojo2-runtime-api'
     implementation group: 'ai.h2o', name: 'mojo2-runtime-impl'
+    constraints {
+        // because https://app.snyk.io/vuln/SNYK-JAVA-COMMONSBEANUTILS-460111
+        compile group: 'commons-beanutils', name: 'commons-beanutils', version: '1.9.4'
```

Comment:
@achraf-mer Did you check the final JARs? Because, sometimes `compil` time constraints doesn't affects the dependency versions in the final JAR